.devcontainer/post-create.sh

```diff
@@ -0,0 +1,23 @@
+# Please note that the following must be set to the name of the
+# repository for the devcontainer to build and run correctly
+REPO_NAME=Trihydro-WyoCV
```

Comment:
I cloned the project without specifying a custom name, so the clone was created with the default name "WyoCV." Although I imported the jpo-ode .jar files, the post-create.sh script failed to locate them because the REPO_NAME variable did not match the name of the cloned directory. It would make sense to update REPO_NAME to reflect the actual repository name.

Reply:
Updated to resolve repository name dynamically

Reply:
Awesome!